libraries/core-react/src/Popover/PopoverItem.tsx

```diff
@@ -0,0 +1,199 @@
+/* eslint-disable @typescript-eslint/no-empty-function */
+import React, { useEffect, useRef, forwardRef, HTMLAttributes } from 'react'
+import styled, { css } from 'styled-components'
+import { Icon } from '../Icon'
+import { Card } from '../Card'
+import { Button } from '../Button'
+import { close } from '@equinor/eds-icons'
+import { spacingsTemplate, typographyTemplate } from '../_common/templates'
+import { useCombinedRefs } from '../_common'
+
+import { popover as tokens, Placement } from './Popover.tokens'
+
+Icon.add({ close })
+
+type WrapperProps = {
+  top: string | number
+  bottom: string | number
+  right: string | number
+  left: string | number
+  transform: string
+}
+
+const StyledPopoverWrapper = styled.div<WrapperProps>`
+  ${({ top, bottom, right, left, transform }) =>
+    css`
+      bottom: ${bottom};
+      top: ${top};
+      right: ${right};
+      left: ${left};
+      transform: ${transform};
+    `}
+  width: max-content;
+  align-self: center;
+  position: absolute;
+  z-index: 500;
+  flex-shrink: 0;
+  ::after {
+    content: '';
+  }
+`
+
+const StyledPopover = styled(Card)`
+  ${typographyTemplate(tokens.header)}
+  ${spacingsTemplate(tokens.spacings)}
+  background: ${tokens.background};
+  fill: ${tokens.background};
+  max-height: ${tokens.popover.maxHeight};
+  max-width: ${tokens.popover.maxWidth};
+  min-height: ${tokens.popover.minHeight};
+  box-shadow: ${tokens.elevation};
+`
+
+type ArrowProps = {
+  top: string
+  bottom: string
+  right: string
+  left: string
+} & HTMLAttributes<SVGSVGElement>
+
+const PopoverArrow = styled.svg<ArrowProps>`
+  ${({ top, bottom, right, left }) =>
+    css`
+      bottom: ${bottom};
+      top: ${top};
+      right: ${right};
+      left: ${left};
+    `}
+  width: ${tokens.arrow.width};
+  height: ${tokens.arrow.height};
+  position: absolute;
+  fill: ${tokens.background};
+  filter: drop-shadow(-4px 0px 2px rgba(0,0,0,0.2));
+`
+
+const StyledCloseButton = styled((props) => <Button {...props} />)`
```

Comment:
Clean code. This can be done as, `styled(Button)`.

Reply:
Done